test/powershell/engine/Job/Jobs.Tests.ps1

```diff
@@ -338,4 +338,48 @@ Describe 'Basic Job Tests' -Tags 'Feature' {
             ValidateJobInfo -job $jobToStop -state 'Stopped' -hasMoreData $false
         }
     }
+
+    Context 'Background pwsh process should terminate after job is done' {
+        It "Can clean up background pwsh process after job is done" {
+            $job = Start-Job { $pid }
+            $processId = Receive-Job $job -Wait
+
+            # The job is done, wait 2 seconds for the cleanup to finish.
+            Start-Sleep -Seconds 2
```

Comment:
Sorry to be nit picky, but you could use the HelpersCommon module 'Wait-UntilTrue' here and loop until Get-Process returns $null.  The tests would probably run quite a bit faster than always waiting 2 seconds.

Reply:
Sure, I will make the change.